src/containers/PipelineRuns/PipelineRuns.test.js

```diff
@@ -362,3 +363,44 @@ it('An invalid filter value is disallowed and reported', async () => {
     )
   ).toBeTruthy();
 });
+
+it('TaskTree handles rerun event in PipelineRuns page', async () => {
+  const mockTestStore = mockStore(testStore);
+  const match = {
+    params: {},
+    url: '/pipelineruns'
+  };
+
+  const onSelect = jest.fn();
```

Comment:
this doesn't appear to be used, the only reference is `onSelect.mockClear()` at the end of the test